partiql/benches/bench_eval_multi_like.rs

```diff
@@ -0,0 +1,425 @@
+use std::time::Duration;
```

Comment:
Maybe a short description of the bench (and the query it benchmarks can also help like `QUERY_15` or `QUERY_30`).